shellwhat/checks.py

```diff
@@ -1,8 +1,19 @@
 import re
-from functools import partial
+from functools import partial, wraps
 from protowhat.checks.check_logic import *
 from protowhat.checks.check_simple import *
 
+ANSI_REGEX = "(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]"
+
+def _strip_ansi(result):
+    return re.sub(ANSI_REGEX, '', result)
+
+def strip_ansi(state):
```

Comment:
`strip_ansi` is exposed as an SCT, like `sort_rows()` in `sqlwhat`. I'm not sure it will be useful, but down the road it seems like it could be useful to let people just strip the ANSI codes once